js/PulsarSortableComponent.js

```diff
@@ -145,4 +163,39 @@ PulsarSortableComponent.prototype.updateOrder = function() {
     });
 };
 
+// Create or update "Actions" badge when a table row is selected
+PulsarSortableComponent.prototype.actionsBadge = function() {
+    var component = this,
+        checkedBoxesCount = component.$html.find('.has-badges table tr input.checkbox:checked').length,
+        badge = component.$html.find('.has-badges .btn__group.dropdown span.badge'),
+        itemWording = component.$html.find('.has-badges .btn__group.dropdown .item-wording'),
+        deleteOption = '.has-badges .dropdown__menu li:last-child a';
+
+    if (checkedBoxesCount > 0) {
```

Comment:
We have an else if, for the case it is zero, in that one. I think we should leave it as is